### ISSUE TITLE
Facial Emotion Detection

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#51]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Facial Emotion Detection</br>
:red_circle: **Aim** : Create a DL model which will detect the emotions from the facial activities.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/mhantor/facial-expression</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by fykaa** [comment]
Full name : Fyka Ansari
GitHub Profile Link : https://github.com/fykaa
Email ID : faeka6@gmail.com
Participant ID (if applicable): na
Approach for this Project : I'll try using a variety of DL techniques and try to do my best. I have little experience with Deep Learning, so it may take some time to implement. Essentially, will use tensorflow and connect with peers to learn. Hopefully, my interest lies around this issue, will be happy if I'm able to complete it.
What is your participant role? Social Summer of Code Participant

**Comment by LahariBoni** [comment]
Hello @fykaa, assigned this issue to you. Experiment with different approaches and include your insights. Hope you learn as much as possible while working on this issue. Happy contribution😊🙌🏻

**Comment by abhisheks008** [comment]
@fykaa updates please.

**Comment by fykaa** [comment]
Working on this, quite tough for me:/

**Comment by abhisheks008** [comment]
Take your time and try to learn more and more from different resources @fykaa

**Comment by abhisheks008** [comment]
Any updates on your learning and project @fykaa

**Comment by abhisheks008** [comment]
@fykaa updates please.

**Comment by Avishkar30** [comment]
Full name : Kadu Avishkar Vasant
    GitHub Profile Link : https://github.com/Avishkar30
    Email ID : kaduavva19ie@student.mes.ac.in
    Participant ID (if applicable): na
   Approach for this Project :  I will use Tensorflow, Keras and openCV using CNN to identify emotions and we will classify 7 types  
    of emotions

**Comment by abhisheks008** [comment]
Issue assigned to you @Avishkar30

**Comment by Avishkar30** [comment]
Hello Sir, 
We have completed the implementation of project 
You can review it on this link :- https://github.com/Avishkar30/Face-Emotion-Recognition

And please tell us what will be further process if required 

Thank You

**Comment by abhisheks008** [comment]
Please follow the project structure and represent your files properly. You need to create a sub folder as per the issue name. Inside that, there will be three folders, Model, Dataset and Images. Another file will be there named as requirements.txt. Put all the files to their respective folders, then only it'll be a complete one. For your reference you can check out the other projects present here.

@Avishkar30

**Comment by abhisheks008** [comment]
Issue has been un-assigned from you due to inactivity.
@Avishkar30

**Comment by stiwari-ds** [comment]
* Full name : **Siddhant Tiwari**
* GitHub Profile Link : https://www.github.com/stiwari-ds
* Email ID : siddhant.tiwari.ds@gmail.com
* Participant ID (if applicable): **s_tiwari** (Devfolio)
* Approach for this Project : 
    1. Different configurations of Convolutional Neural Networks (Manual + Hyperparameter tuning)
    2. Implement transfer-learning techniques for improving accuracy
    3. Experiment with transformer models from recent research
* What is your participant role? (Mention the Open Source program): Individual Contributor, SWOC-S3 (2023)

**Comment by abhisheks008** [comment]
Issue assigned to you @stiwari-ds

**Comment by Praneesh-Sharma** [comment]
Full name : Praneesh Sharma
GitHub Profile Link : https://github.com/Praneesh-Sharma
Email ID :praneeshsharma30@gmail.com
Participant ID (if applicable):
Approach for this Project : I will start with applying a basic CNN model, and then move on to more complex alternatives based on the performance of the model.
What is your participant role? (Mention the Open Source program): Individual Contributor, SSOC-S2(2023)
